bin/indexGenerator.py

```diff
@@ -148,6 +153,18 @@ def generate_footer(self):
         with open(self.footer, "w") as f:
             f.write(template.render(contexts))
 
+    def fetch_pubkeyinfo(self):
+        pub_key_info = ""
+
+        if (self.gpg_pub_key_info_file != "."):
+            gpg_pub_key = Path(self.gpg_pub_key_info_file)
+            if (gpg_pub_key.is_file()): 
+                gpg_pub_key = open(self.gpg_pub_key_info_file, "r")
+                pub_key_info = gpg_pub_key.read()
+                gpg_pub_key.close()
```

Comment:
Allow python to handle closing the file?
```suggestion
                with open(self.gpg_pub_key_info_file, "r") as gpg_pub_key:
                    pub_key_info = gpg_pub_key.read()
```